homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -276,7 +276,9 @@ async def async_service_handler(service: ServiceCall) -> None:
                 continue
             await entity_method(**params)
             update_tasks.append(
-                hass.async_create_task(entity.async_update_ha_state(True))
+                hass.async_create_task(
+                    asyncio.create_task(entity.async_update_ha_state(True))
```

Comment:
I dont think this is supposed to be 2x asyncio.create_task

Reply:
Good point! Fixed.